config/initializers/monkeypatches/object.rb

```diff
@@ -1,10 +1,12 @@
 module PresenceBangMonkeypatch
+  class ObjectNotPresent < StandardError ; end
```

Comment:
Adding this error class was partially motivated by a desire to make it easier to test for this error to be raised, but I think it's probably also a good idea, in general, to have this be a dedicated error class.